### PR TITLE
Revert CRDs move to v1, move back to v1beta1

### PR DESCRIPTION
**Pull request by nikolay-pshenichny**
This change reverts #25263. It is the last step of a series of PRs that will move back to using v1beta1 APIs to add back support for kubernetes 1.15.

First PR - https://github.com/istio/tools/pull/1138
Second PR - https://github.com/istio/api/pull/1602

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

@joeyb , @ramaraochavali

### CODE REVIEW

File: bin/update_crds.sh [hunk 1]
```diff
@@ -43,11 +43,13 @@ if [ -z "${SHA}" ]; then
...
 
-git clone  "https://${REPO}" "${API_TMP}" && cd "${API_TMP}"
-git checkout "${SHA}"
-if [ ! -f "${API_TMP}/kubernetes/customresourcedefinitions.gen.yaml" ]; then
-  echo "Generated Custom Resource Definitions file does not exist in the commit SHA ${SHA}. Not updating the CRD file."
-  exit
-fi
-rm -f "${ROOTDIR}/manifests/charts/base/crds/crd-all.gen.yaml"
-cp "${API_TMP}/kubernetes/customresourcedefinitions.gen.yaml" "${ROOTDIR}/manifests/charts/base/crds/crd-all.gen.yaml"
+echo "NIKO: skipping crds update"
```
Comment (nikolay-pshenichny) [inline bin/update_crds.sh:46]: TODO: need to revert this when the other PRs are merged

**Comment by howardjohn** [comment]
Looks good overall. There is one real failure: `panic: interface conversion: proto.Message is *v1beta1.CustomResourceDefinitionSpec, not *v1.CustomResourceDefinitionSpec` in the pilot test.

We need to do manual testing to ensure users can gracefully go from 1.6 -> 1.7 (beta->v1) then 1.7 -> 1.8 (v1->beta)

**Comment by nikolay-pshenichny** [comment]
> Looks good overall. There is one real failure: panic: interface conversion: proto.Message is *v1beta1.CustomResourceDefinitionSpec, not *v1.CustomResourceDefinitionSpec in the pilot test.

@howardjohn, I updated `galley/pkg/config/analysis/analyzers/deprecation/deprecation.go`, the error should be gone now. please take a look - https://github.com/istio/istio/pull/26587/commits/2a2ba042ca2467454f9e00c00db3871e3250c7c0

**Comment by nikolay-pshenichny** [comment]
@howardjohn, 

We did some basic validation  of 1.7 to master upgrade on k8s 1.16 using docker-for-desktop. The demo app continues to work post the upgrade. 
Will it be possible to merge https://github.com/istio/tools/pull/1138, so that we can update other PRs ? 
Thank you!

**Comment by howardjohn** [comment]
@nikolay-pshenichny thanks! will approve that one in a minute. Feel free to mention me on the relevant other PRs when needed

**Comment by nikolay-pshenichny** [comment]
@howardjohn , I updated this PR with the latest master. Please take a look.

**Review by howardjohn — APPROVED** [review]

**Review by jasonwzm — APPROVED** [review]

**Pull request merged**